specification/glossary.md

```diff
@@ -2,10 +2,130 @@
 
 *This section is non-normative.*
 
-`TODO` Needs to be completed and linked
+## Adjustment
 
-Resource:
-> Add definition
+A charge representing a modification to billing data to account for certain events or circumstances not previously captured, or captured incorrectly. Examples include: billing errors, service disruptions, or pricing changes.
 
-Service:
-> Add definition
+## Amortization
+
+The gradual recognition of upfront costs over the duration of time that the good or service is consumed. An upfront payment made toward a commitment based discount can be amortized over commitment life. Amortization is valuable when the commitment period (time duration of the cost) extends beyond the granularity of the source report.
+
+## Billed Cost
+
+The total amount of fees and discounts as documented in an itemized bill.
+
+## Billing Account
+
+A mandatory account established with a service provider to manage and pay for all charges associated with an invoice. A billing account might have sub-accounts, all of whose costs are consolidated to the billing account.
```

Comment:
From BillingAccountId/Name: 
"A billing account is a container for resources and/or services that are billed together in an invoice."

The initial thinking was that we'd use the definition from the spec (don't have to if we believe this is better). I definitely like the second sentence. Should it say 
"all of whose costs are consolidated to and invoiced at the billing account."